Makefile

```diff
@@ -113,6 +113,35 @@ install: compile
 	done
 	$(MAKE) install_man
 
+check_reproducible: compile
+	$(Q) echo "==> Checking for reproducible builds..."
+	export SOURCE_DATE_EPOCH="$(shell bin/elixir -e \
+                                           'IO.puts System.build_info()[:date] \
+                                            |> DateTime.from_iso8601() \
+                                            |> elem(1) \
+                                            |> DateTime.to_unix()')"
+	$(Q) $(MAKE) clean_reproducible_files
+	$(Q) echo "Moving files to ebin_reproducible/ ..."
+	$(Q) mkdir -p lib/elixir/tmp/ebin_reproducible/ \
+                      lib/eex/tmp/ebin_reproducible/ \
+	              lib/iex/tmp/ebin_reproducible/ \
+	              lib/logger/tmp/ebin_reproducible/ \
+	              lib/mix/tmp/ebin_reproducible/
+	$(Q) mv lib/elixir/ebin/* lib/elixir/tmp/ebin_reproducible/
+	$(Q) mv lib/eex/ebin/* lib/eex/tmp/ebin_reproducible/
+	$(Q) mv lib/iex/ebin/* lib/iex/tmp/ebin_reproducible/
+	$(Q) mv lib/logger/ebin/* lib/logger/tmp/ebin_reproducible/
+	$(Q) mv lib/mix/ebin/* lib/mix/tmp/ebin_reproducible/
+	SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH}" $(MAKE) compile
+	$(Q) echo "Diffing..."
+	$(Q) diff -r lib/elixir/ebin/ lib/elixir/tmp/ebin_reproducible/
+	$(Q) diff -r lib/eex/ebin/ lib/eex/tmp/ebin_reproducible/
+	$(Q) diff -r lib/iex/ebin/ lib/iex/tmp/ebin_reproducible/
+	$(Q) diff -r lib/logger/ebin/ lib/logger/tmp/ebin_reproducible/
+	$(Q) diff -r lib/mix/ebin/ lib/mix/tmp/ebin_reproducible/
+	$(Q) $(MAKE) clean_reproducible_files
```

Comment:
We don't need this call now (nor the task) as we do leave dirty stuff in `tmp`. That's why it exists! :D